utils.ml

```diff
@@ -155,16 +155,19 @@ let button_component ~attribs ~content ~btn_type ?(extra_css = "") () =
 
 (** A UI with two buttons, one for decrementing a value and the other for
     incrementing the value*)
-let increment_or_decrement_ui ~max_value ~min_value ~figure_unit ~id ~label' =
+let increment_or_decrement_ui ~max_value ~min_value ?(default_value = 0)
+    ?(figure_unit = "") ~id ~label' () =
   Tyxml_html.(
```

Comment:
so you never check that max is bigger than min?

what if min is -10, max 10, and default should indeed be 0? I suggest to have no default value for default to distinguish the valid use of 0 from nothing supplied